confgenerator/testdata/valid/linux/logging-receiver_cassandra/golden_fluent_bit_parser.conf

```diff
@@ -0,0 +1,224 @@
+[PARSER]
```

Comment:
Any Cassandra specific invalid user config we need to test against?

Reply:
Nothing I could find.